.github/workflows/coverage-report.yml

```
@@ -0,0 +1,95 @@
+          script: |
+            const fs = require('fs');
+            const path = require('path');
+
+            // Read coverage summary
+            const coveragePath = path.join('apps', 'express-api', 'coverage', 'coverage-summary.json');
+            if (fs.existsSync(coveragePath)) {
+              const coverage = JSON.parse(fs.readFileSync(coveragePath, 'utf8'));
+              const total = coverage.total;
+              
+              const comment = `## 📊 Coverage Report
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)